src/routes/cluster.routes.js

```diff
@@ -0,0 +1,13 @@
+module.exports = app => {
+  const clusters = require("../controllers/cluster.controller.js");
+
+  app.post("/clusters", clusters.create);
```

Comment:
I prefer a better namespaced route to contain `/api/<VERSION>/<THING>/`. Up for the discussion but would like to see things conform to this standard